example/test/support/query_helper.exs

```diff
@@ -0,0 +1,24 @@
+defmodule QueryHelper do
```

Comment:
I'm sorry to ask for this, but I think this abstraction makes are tests hard to read with very little benefit, could you remove this abstraction and change the tests?